test/e2e/crcsuite/crcsuite.go

```diff
@@ -433,7 +435,11 @@ func CommandReturnShouldContainIfBundleEmbeddedOrNot(commandField string, value
 func SetConfigPropertyToValueSucceedsOrFails(property string, value string, expected string) error {
 	if value == "current bundle" {
 		if bundleEmbedded {
-			value = filepath.Join(CRCHome, "cache", bundleName)
+			if installerBuild != "false" {
+				value = filepath.Join(cmd.GetCRCBinaryPath(), bundleName)
```

Comment:
That mean when it is not the installer build then the bundle location should be same as binary location? Also I am not sure why we are making this change on master now (this should be just maintenance branch for time being).

Reply:
Well this is the current behavior managed not based on platform but on the installer control variable, But basically installers uses its own path (os dependant) in the meantime linux uses old home folder...so just fixing this as this is creating a false postive on e2e tests.

THis is done on master as it is fixing a bug on e2e.

Reply:
Ack